pkgs/objective_c/tool/coverage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Nit: In general I prefer Dart scripts, dart works fine as a scripting language. Feel free to leave as bash script. But as soon as scripts start growing we should make things Dart scripts. (the fact that we have two scripts that are slightly similar but not exactly makes me feel they could be shared in a toplevel tool dir that is used by both pkgs/ffigen and pkgs/objective_c)